oak/server/oak_grpc_node.cc

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2018 The Project Oak Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "oak/server/oak_grpc_node.h"
+#include "absl/memory/memory.h"
+#include "asylo/grpc/auth/enclave_server_credentials.h"
+#include "asylo/grpc/auth/null_credentials_options.h"
+#include "asylo/util/logging.h"
+#include "include/grpcpp/grpcpp.h"
+#include "oak/server/module_invocation.h"
+
+namespace oak {
+
+std::unique_ptr<OakGrpcNode> OakGrpcNode::Create() {
+  std::unique_ptr<OakGrpcNode> node = absl::WrapUnique(new OakGrpcNode());
```

Comment:
Does `absl::make_unique` work here?

Reply:
Done

Reply:
Actually reverted this, we need WrapUnique because the constructor is private